app/main/controllers/projects.py

```diff
@@ -46,6 +46,27 @@ def create_and_insert_source(url, project_id, x=None, y=None):
     return source
 
 
+def create_and_insert_item(content, project_id, source_id, x=None, y=None):
+    # note and highlight booleans
+    is_highlight = check_if_highlight(content)
```

Comment:
I think we talked about this, but you can assume that you'll receive something like an `is_note` parameter from the API call, so you wouldn't need to use this function (and also wouldn't need to set `is_note` to be the opposite of `is_highlight`, since that adds redundancy)

Reply:
Again, will take out is_highlight soon, and also removed the whole function check_if_highlight in general, since Daniel will be passing the booleans from the frontend